compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -161,8 +161,7 @@ pub unsafe fn create_module<'ll>(
     // Ensure the data-layout values hardcoded remain the defaults.
     if sess.target.is_builtin {
         let tm = crate::back::write::create_informational_target_machine(tcx.sess);
```

Comment:
Could you add a comment here saying that the machine is disposed by the drop impl?

Reply:
I'd hope that would be clear from the type that is returned.
But I can do it anyways.

Reply:
This code mixes manual and automatic resource management and calling the LLVM API in funny ways, so I think being explicit makes sense.